docs/src/guide/load-balancer-annotations.md

```diff
@@ -0,0 +1,147 @@
+# Load Balancer Annotations
+
+This file defines a list of [Service][4] `type: LoadBalancer` annotations which are
```

Comment:
Don't just copy from the OCI CCM docs. This will be published to github pages and not accessed by file. Rework the docs with publishing in mind.